notices.json

```diff
@@ -1,4 +1,24 @@
 [
+    {
+    "id": "june15-cloud-freemium",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "sku": "e0",
+      "instanceType": "onprem",
+      "repeatable": true,
```

Comment:
This flag means all our on-prem users will see it everyday for three days, so it may be overly disruptive.

I'm also not sure how well the repeatable flag has been tested. Could I propose that we don't use that flag, and if we want to refire the notice sometime in the future we just submit a new one?

Reply:
:+1: sounds good, let's do that. Removed the flag